simulation/simulation_interface/proto/std_msgs.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+import "builtin_interfaces.proto";
+package std_msgs;
+
+/**
+ * Protobuf definiciton of [std_msgs::msgs::Header type in
+ *ROS2.](https://github.com/ros2/common_interfaces/blob/master/std_msgs/msg/Header.msg)
+ **/
```

Comment:
Broken indentation

Reply:
fixed in f6866faa0ee81aa8172e74b4322c48adfbbce10e

Reply:
fixed in 082e52f817ed4a829eeddfdd9148cd0455d8661b